app/services/green_lanes/determine_next_page.rb

```diff
@@ -0,0 +1,49 @@
+module GreenLanes
+  class DetermineNextPage
+    def initialize(goods_nomenclature)
+      @categories = GreenLanes::DetermineCategory.new(goods_nomenclature).categories
+    end
+
+    def next(cat_1_exemptions_apply: nil, cat_2_exemptions_apply: nil)
+      case @categories
+      when [:cat_1], [:cat_2], [:cat_3]
+        # Simplest case: only one category is present
+        "result_#{@categories.first}".to_sym
```

Comment:
will this always be in the same order?

Reply:
@categories is an array containing the :cat_1, :cat_2, :cat_3